rwandaemr-installer/src/main/resources/scripts/vars.txt

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+## Distribution can be rwandaemr-imb-butaro or rwandaemr-imb-rwinkwavu
+distribution="rwandaemr-imb-butaro"
+## Version can either be 1.x or 2.x
+version="1.1.0-SNAPSHOT"
+## Any directory that you'd like your environment to be installed in
+target_dir="butaro"
```

Comment:
This "target_dir" is very misleading and I don't really get it.  This isn't actually ever created as either the TARGET_DIR, or any directory at all.  It is just a component that is used within a TARGET_DIR and USER.  I'd call this something else that reflects the kinds of things you intend for it to represent.

Reply:
How would you call this destination_dir?

Reply:
I'd call this "site" or "implementation" or "base_name" or something, not sure.  But it isn't a directory and it doesn't reflect the ultimate name of a directory - it's used to create a sub directory name within the target directory.